build.sbt

```diff
@@ -270,6 +270,14 @@ lazy val server = (project in file("server"))
       "com.h2database" %  "h2" % "2.2.224",
 
       "org.hibernate.orm" % "hibernate-core" % "6.5.0.Final",
+      // For loading properties from the environment
+      "uk.org.webcompere" % "lightweight-config" % "1.3.0",
+
+      // For MySQL backend
+      "com.mysql" % "mysql-connector-j" % "8.4.0",
```

Comment:
Hey @xx-db 👋 I saw you tried to add the mysql jdbc driver back in #211, but ultimately it wasn't merged. Currently the CI is complaining about a license issue by adding it here. Was that what you encountered? Do you know who might be able to either:

a) Approve overriding the license lint, and allow the connector to be used
b) Confirm that we can't use the connector due to a license conflict, and therefore the project should stop mentioning using MySQL

Thanks in advance!

CC: @dennyglee

Reply:
@tnightengale 👋You are right, it was ultimately not merged because they are commonly used jars supported by Hibernate. So we included documentation to download the connector jars instead.